cmd/minishift/cmd/addon/addon_disable.go

```diff
@@ -44,16 +44,18 @@ func runDisableAddon(cmd *cobra.Command, args []string) {
 		atexit.ExitWithMessage(1, emptyDisableError)
 	}
 
-	addonName := args[0]
+	addOnName := args[0]
```

Comment:
@budhrg I think `add` `on` is not a single word. Also in other places , variables and functions are named as `addOn`* so I made it consistent.